apps/sms/style/root.css

```diff
@@ -79,6 +79,31 @@ small {
   color: #999;
 }
 
+/* Tabs */
```

Comment:
The styling here is mostly for edit form, but we don't put these in root.css because it's for genaral element. Please put them in sms.css instead